metals/src/main/scala/scala/meta/internal/worksheets/WorksheetProvider.scala

```diff
@@ -313,9 +313,9 @@ class WorksheetProvider(
   private def getMdoc(target: BuildTargetIdentifier): Option[Mdoc] = {
 
     def isSupportedScala3Version(scalaVersion: String) = {
-      // Worksheet support for Scala 3 started with 0.26.0-RC1
+      // Worksheet support for Scala 3 is only working currently for 3.0.0-M1
       ScalaVersions.isScala3Version(scalaVersion) && SemVer.isCompatibleVersion(
-        "0.26.0",
+        "3.0.0",
```

Comment:
So I was curious what would happen if I was still no 0.27, and this is shown in the logs
```
2020.11.06 13:03:03 WARN  worksheet: unsupported Scala version '0.27.0-RC1', using Scala version '2.12.12' without classpath instead.
worksheet: to fix this problem use Scala version '3.0.0-M1'.
```
We don't have to do this now, but what do you think about actually floating this up to the user? I think it might be confusing for someone since I'm not sure I'd expect it to revert to 2.12.12 without the classpath. It may cause confusion.

Reply:
Yeah, it might be good to do, I will try to follow up with that and merge this so that people can start testing out 3.0.0-M1